x-pack/plugins/security_solution/common/asset_criticality/constants.ts

```diff
@@ -5,9 +5,65 @@
  * 2.0.
  */
 
+import { i18n } from '@kbn/i18n';
+import { euiLightVars } from '@kbn/ui-theme';
+import type { AssetCriticalityRecord } from '../api/entity_analytics/asset_criticality';
+
 export const ASSET_CRITICALITY_INDEX_PATTERN = '.asset-criticality.asset-criticality-*';
 
 type AssetCriticalityIndexPrivilege = 'read' | 'write';
 export const ASSET_CRITICALITY_REQUIRED_ES_INDEX_PRIVILEGES = {
   [ASSET_CRITICALITY_INDEX_PATTERN]: ['read', 'write'] as AssetCriticalityIndexPrivilege[],
 };
+
+export const PICK_ASSET_CRITICALITY = i18n.translate(
```

Comment:
Not the best place, as a general rule of thumb:

- `security_solution/common/*` -> shared between `public` and `server` directories (URLs, indexPatterns, privileges..)
- `security_solution/public/common/*` -> shared between `public` "sub-plugin" directories (reusable components, reusable hooks, paths, constants...)

These translations, however, do not seem to be _common_ stuff, they are only used in the `security_solution/public/entity_analytics/components/asset_criticality_selector.tsx` component. 

So, it would be better to put them in a new _translations_ file inside the same directory, like: `security_solution/public/entity_analytics/components/translations.ts`. 

However, how things are organized inside the `security_solution/public/entity_analytics` directory is up to your team.

Reply:
`security_solution/public/entity_analytics/components/translations.ts` is a good option. The docs recommend inlining the copy into the react component using `<FormattedMessage>`.

You can read more here: https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md

Reply:
I moved stuff around, most of this stuff now lives along side the component in a new asset criticality folder within EA